Assets/MixedRealityToolkit-SDK/Features/Utilities/Solvers/SolverHandler.cs

```diff
@@ -11,8 +11,7 @@ namespace Microsoft.MixedReality.Toolkit.SDK.Utilities.Solvers
     /// <summary>
     /// This class handles the solver components that are attached to this <see cref="GameObject"/>
     /// </summary>
-    [DisallowMultipleComponent]
-    public class SolverHandler : MonoBehaviour
+    public class SolverHandler : ControllerFinder
```

Comment:
If anything we should be inheriting form the pose synchronizer.

Reply:
As I mentioned in the PR where you created that pose synchronizer, it does not fulfill the requirements of the original ControllerFinder. It serves a similar purpose, but is incompatible with this case.

Reply:
Those requirements being?